packages/programs/src/program.ts

```diff
@@ -0,0 +1,34 @@
+import type { Address } from '@solana/addresses';
+
+/**
+ * Defines a Solana program.
+ */
+export type Program<TAddress extends string = string> = {
+    /**
+     * A unique name for the Program.
+     *
+     * To avoid conflict with other organizations, it is recommended
+     * to prefix the program name with a namespace that is unique to
+     * your organization. For instance, programs belonging to the
+     * Solana Program Library are prefixed with `spl` like so:
+     * `splSystem` or `splAddressLookupTable`.
```

Comment:
nit: These aren't SPL programs, so they don't actually have these prefixes. Maybe use SPL Memo or SPL Token?

Reply:
Good shout, I'll update first thing in the morning so it's less confusing.